components/predict-form/question-types/MultiChoiceQuestion.tsx

```diff
@@ -60,24 +77,34 @@ export default function MultiChoiceQuestion({
           />
         ))}
 
-        {fields.length < MAX_OPTIONS && (
```

Comment:
Might be wrong, but I think this is now handled by the addOption callback, decided to remove but let me know if I've missed something